pkgs/applications/misc/dunst/default.nix

```diff
@@ -31,7 +31,10 @@ stdenv.mkDerivation rec {
     "SERVICEDIR_SYSTEMD=$(out)/lib/systemd/user"
   ];
 
+  buildFlags = [ "dunstify" ];
+
   postInstall = ''
+    install -Dm755 dunstify $out/bin
```

Comment:
This binary should also be wrapped with `GDK_PIXBUF_MODULE_FILE`, no?

Reply:
No, dunstify is just a CLI client for the private functionality of dunst which is exposed by D-Bus.

Reply:
I tested and I can call `dunstify` with `--icon`, which would be the only place I'd expect pixbuf to be used.